code/game/objects/items/oddities.dm

```diff
@@ -102,6 +100,7 @@
 	name = "old newspaper"
 	desc = "It contains a report on some old and strange phenomenon. Maybe it's lies, maybe it's corporate experiments gone wrong."
 	icon_state = "old_newspaper"
+	//perk = PERK_ASS_OFF_CONCRETE
```

Comment:
Those perks must be provided to oddities at random, so there is no need to do that.

There is no need to do that in PR for different perks anyway.

Reply:
uhm. I thought I had deleted this